src/components/ToolMenu/Draw/ColorPickDrawer/index.less

```diff
@@ -0,0 +1,6 @@
+.color-pick-drawer {
+  position: absolute;
+  top: 50px;
+  bottom: 40px;
```

Comment:
We might want to include the `@headerHeight` and `@footerHeight` variables from [here](https://github.com/terrestris/shogun-gis-client/blob/main/src/variables.less).